templates/articles/blocks/featured_link.html

```diff
@@ -53,10 +53,18 @@
                             By {{ value.author }}
                         </p>
                     {% endif %}
-        
+
+                    {% if value.publication_date %}
+                        <p class="featured-link__metadata-item">
+                            {{ value.publication_date }}
+                        </p>
+                    {% endif %}
+                    
+                    {% comment %} TODO: Un-comment this when blogs go live properly and we use a proper date type
                     {% if value.publication_date %}
                         <time datetime="{{ value.publication_date|date:'Y-m-d' }}" class="featured-link__metadata-item">{{ value.publication_date|date:'l j F Y' }}</time>
                     {% endif %}
```

Comment:
@jamesbiggs I think we should just go ahead and delete the old code instead of commenting it out. No doubt you'll be using completely different blocks when blog posts are pages